packages/block-editor/src/components/inner-blocks/README.md

```diff
@@ -180,3 +180,8 @@ For example, a button block, deeply nested in several levels of block `X` that u
 
 -   **Type:** `Function`
 -   **Default:** - `undefined`. The placeholder is an optional function that can be passed in to be a rendered component placed in front of the appender. This can be used to represent an example state prior to any blocks being placed. See the Social Links for an implementation example.
+
+### `prioritizedInserterBlocks`
```

Comment:
do we need to document this somewhere else or is this enough?

Reply:
That's the official docs so I guess that's enough.